deploy/crds/shipwright.io_buildruns.yaml

```diff
@@ -650,6 +650,13 @@ spec:
                           image:
                             description: Image reference, i.e. quay.io/org/image:tag
                             type: string
+                          prune:
+                            description: "Prune specifies whether the image is suppose
+                              to be deleted. Allowed values are 'Never' (no deletion)
+                              and `AfterPull` (removal after the image was successfully
+                              pulled from the registry). \n If not defined, it defaults
+                              to 'Never'."
+                            type: string
```

Comment:
What do you think about possible alternatives. I actually started with an iota style const enumeration, or whatever it is correctly called in Go. But, I did not like that it would be an intrinsic type like an int. So I settled for a string instead.